File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL0000Q00GetSigeyulDataSelect2ListItemInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL0000Q00GetSigeyulDataSelect2ListItemInfo {
	private String cplResult;
    private String standardYn;
    private String gumsaName;
    private String fromStandard;
    private String toStandard;
	public CPL0000Q00GetSigeyulDataSelect2ListItemInfo(String cplResult,
			String standardYn, String gumsaName, String fromStandard,
			String toStandard) {
		super();
		this.cplResult = cplResult;
		this.standardYn = standardYn;
		this.gumsaName = gumsaName;
		this.fromStandard = fromStandard;
		this.toStandard = toStandard;
	}
	public String getCplResult() {
		return cplResult;
	}
	public void setCplResult(String cplResult) {
		this.cplResult = cplResult;
	}
	public String getStandardYn() {
		return standardYn;
	}
	public void setStandardYn(String standardYn) {
		this.standardYn = standardYn;
	}
	public String getGumsaName() {
		return gumsaName;
	}
	public void setGumsaName(String gumsaName) {
		this.gumsaName = gumsaName;
	}
	public String getFromStandard() {
		return fromStandard;
	}
	public void setFromStandard(String fromStandard) {
		this.fromStandard = fromStandard;
	}
	public String getToStandard() {
		return toStandard;
	}
	public void setToStandard(String toStandard) {
		this.toStandard = toStandard;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL0000Q00GetSigeyulDataSelect2ListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL0000Q00GetSigeyulDataSelect2ListItemInfo
	{
		private String _cplResult;
		private String _standardYn;
		private String _gumsaName;
		private String _fromStandard;
		private String _toStandard;

		public String CplResult
		{
			get { return this._cplResult; }
			set { this._cplResult = value; }
		}

		public String StandardYn
		{
			get { return this._standardYn; }
			set { this._standardYn = value; }
		}

		public String GumsaName
		{
			get { return this._gumsaName; }
			set { this._gumsaName = value; }
		}

		public String FromStandard
		{
			get { return this._fromStandard; }
			set { this._fromStandard = value; }
		}

		public String ToStandard
		{
			get { return this._toStandard; }
			set { this._toStandard = value; }
		}

		public CPL0000Q00GetSigeyulDataSelect2ListItemInfo() { }

		public CPL0000Q00GetSigeyulDataSelect2ListItemInfo(String cplResult, String standardYn, String gumsaName, String fromStandard, String toStandard)
		{
			this._cplResult = cplResult;
			this._standardYn = standardYn;
			this._gumsaName = gumsaName;
			this._fromStandard = fromStandard;
			this._toStandard = toStandard;
		}

	}
}